### PR TITLE
Cross compiling for netstandard1.5 & net46

### PR DESCRIPTION
**Pull request by vijayrkn**
@ahmelsayed

### CODE REVIEW

File: src/Microsoft.NET.Sdk.Functions/MakeFunction/PathUtility.cs [hunk 1]
```diff
@@ -20,7 +20,11 @@ internal static string MakeRelativePath(string fromPath, string toPath)
             Uri relativeUri = fromUri.MakeRelativeUri(toUri);
             String relativePath = Uri.UnescapeDataString(relativeUri.ToString());
 
+#if NET46
             if (toUri.Scheme.Equals("file", StringComparison.InvariantCultureIgnoreCase))
+#else
+            if (toUri.Scheme.Equals("file", StringComparison.OrdinalIgnoreCase))
```
Comment (ahmelsayed) [inline src/Microsoft.NET.Sdk.Functions/MakeFunction/PathUtility.cs]: We can just use OrdinalIgnoreCase in NET46 as well.

**Review by ahmelsayed — COMMENTED** [review]
do the dlls in references have to be checked into the repo? Can't they be downloaded using nuget?

File: src/Microsoft.NET.Sdk.Functions/MakeFunction/PathUtility.cs (hunk 1 above)
Comment (vijayrkn) [reply src/Microsoft.NET.Sdk.Functions/MakeFunction/PathUtility.cs]: Sure. Will change it.

**Comment by vijayrkn** [comment]
Actually we can take it from the output path itself.. will remove the reference and take it from the output path.

**Review by ahmelsayed — APPROVED** [review]

**Comment by vijayrkn** [comment]
Updated

**Pull request merged**